### PR TITLE
Cymetric Documentation Fixes

### PR DESCRIPTION
**Pull request by scopatz**
This provides much needed cleanup and fixes for the cymetric docs.

### CODE REVIEW

File: source/user/cymetric/tutorial_metrics.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+review a quick tutorial, `10 Minutes to pandas
+<http://pandas.pydata.org/pandas-docs/stable/10min.html>`_.
+
+Making Metrics from Custom Database Tables
+------------------------------------------
+The above shows how easy it is to incorporate metrics that are computed via
+cymetric. Moreover, |cyclus| databases can be comprised of both `default tables
+<http://fuelcycle.org/user/dbdoc.html#table-descriptions>`_ and  `custom tables
+<http://fuelcycle.org/arche/custom_tables.html>`_. Cymetric also helps you
+bring in data that might come a custom table in a |cyclus| database.  All you
```
Comment (FlanFlanagan) [inline source/user/cymetric/tutorial_metrics.rst]: come as a custom table?

Comment (scopatz) [reply source/user/cymetric/tutorial_metrics.rst]: Yesm archetype authors can write their own custom tables: http://fuelcycle.org/arche/custom_tables.html

Comment (FlanFlanagan) [reply source/user/cymetric/tutorial_metrics.rst]: Sorry, to clarify, I was asking if that was a typo or not.

Comment (scopatz) [reply source/user/cymetric/tutorial_metrics.rst]: Oh sorry! It says `come a`.  Yes that is a typo

Comment (scopatz) [reply source/user/cymetric/tutorial_metrics.rst]: Fixed!

**Comment by bam241** [comment]
metric's pages are empty is it expected ? otherwise looks good to me!

thank you @scopatz !

**Comment by scopatz** [comment]
@bam241 which page?

**Comment by FlanFlanagan** [comment]
I think he means the root metric / fco metric pages

**Comment by scopatz** [comment]
Ahhh, if those pages are empty, then that is because there are no docstrings in cymetric itself.  They should be added, but that has to happen in the cymetric repo, not here.

**Comment by FlanFlanagan** [comment]
merged

**Pull request merged**